content/source/docs/enterprise/upgrade/index.html.md

```diff
@@ -97,35 +86,31 @@ The user account that owns this token must be a member of both the new and legac
 - **Read** access to any legacy environments that the configuration needs to read state from.
 - **Write** access to the new workspace where the configuration will run.
 
-If you use this workaround during your migration process, you must do two additional steps after you've migrated all of your legacy environments:
+If you use this workaround during your migration process, you must do two additional steps after migrating all of your legacy environments:
 
 1. Edit this workspace's configuration so it reads state data from the new workspaces that replaced your legacy environments.
 2. At the same time, delete `ATLAS_TOKEN` from this workspace's variables.
 
 Make a note of any workspaces that access remote state as you migrate them, and update them soon after you finish migrating. Leaving a value for `ATLAS_TOKEN` over the long term can make your workspaces unnecessarily fragile.
 
-## 5. Disable VCS Integration in the Legacy Environment
+### Step 3.e: Unlock Workspace and Queue Plan
 
-Before committing and pushing the backend change to VCS, the legacy environment webhook must be disabled to prevent the configuration from running in both the legacy environment and the new workspace.
+Upon creation, the migrated workspace is locked to prevent runs. Unlock it after verifying the settings and variables, and queue a plan.
```

Comment:
As of the auto-locking changes, we also auto-unlock the workspace after the migration has finished.